src/libs/link-expiration-alerts.test.ts

```diff
@@ -11,14 +14,26 @@ jest.mock('src/auth/auth', () => {
   };
 });
 
-jest.mock('src/libs/providers', () => [
-  {
-    key: 'anvil',
-    name: 'NHGRI AnVIL Data Commons Framework Services',
-    expiresAfter: 30,
-    short: 'NHGRI',
-  },
-]);
+jest.mock('src/libs/config', () => ({
+  ...jest.requireActual('src/libs/config'),
+  getConfig: jest.fn().mockReturnValue({}),
+}));
+
+jest.mock('src/profile/external-identities/OAuth2Providers', () => ({
+  allOAuth2Providers: [
+    {
+      key: 'fence',
+      name: 'NHLBI BioData Catalyst Framework Services',
```

Comment:
Is there a reason you changed the test provider from Anvil to Biodata catalyst?

Reply:
AnVIL is going away soon! It'll be handled via passports instead of fence providers!

Reply:
Oh that's right!